app/helpers/application_helper.rb

```diff
@@ -1,12 +1,10 @@
 module ApplicationHelper
-  FEEDBACK_EMAILS = {
-    lambeth: "digitalplanning@lambeth.gov.uk",
-    southwark: "digital.projects@southwark.gov.uk",
-    buckinghamshire: "planning.digital@buckinghamshire.gov.uk",
-  }.freeze
-
   def feedback_email(current_local_authority)
-    FEEDBACK_EMAILS[current_local_authority.to_sym]
+    I18n.t("local_authorities.#{current_local_authority}.feedback_email")
+  end
+
+  def privacy_notice(current_local_authority)
+    I18n.t("local_authorities.#{current_local_authority}.privacy_policy")
```

Comment:
I agree with putting the email and url in a yml file. But since they aren't technically translations I'm not 100% sure it makes sense to put them in `en.yml` and use `I18n.t` to retrieve them. It's not like they'd change if the site was in welsh!

There's some local-authority specific information in `'config/contact_details.yml'`, which is then retrieved [here](https://github.com/unboxed/bops-applicants/blob/main/app/helpers/application_helper.rb#L25) - would that be a better place for them?

Reply:
Moved it, that's a great suggestion! Thanks